### PR TITLE
Small fixes to developer documentation

### PR DESCRIPTION
**Pull request by jscheidtmann**
- Provide a howto in the "Documentation" secion, how to compile and review documentation locally.
 - Added note that development on windows is possible.

### CODE REVIEW

File: docs/source/dev_guide.rst [hunk 1]
```diff
@@ -54,16 +54,21 @@ the procedure. If not, here is how you do this:
...
 
 The `PiFinder documentation <https://pifinder.readthedocs.io/en/release/index.html>`_
 is written in `reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links>`_ . 
-The files are located in PiFinders GitHub repository under ``docs`` and have 
-the ending ``.rst``. Many open source projects use `redthedocs.io <eadthedocs.io>` 
-for creating documentation since it is immediately generated, when you are committing 
-the GitHub code (CI/CD pipeline). It is very easy to link your fork of the documentation 
-code to GitHub. 
+The files are located in PiFinders GitHub repository under ``docs/source`` and have 
+the ending ``.rst``. We publish the documentation to `redthedocs.io <eadthedocs.io>`_ manually.
```
Comment (brickbots) [inline docs/source/dev_guide.rst:62]: Thanks for adding the section on previewing docs locally, it's very useful!   The information about how the documentation is generated is probably a bit unclear, but it is automatic.

Let's revert the change above here and I'll clarify a little bit about how this works as my comment about the CI/CD pipeline and fork linkage is pretty misleading :+1:

File: docs/source/dev_guide.rst [hunk 2]
```diff
@@ -89,6 +94,10 @@ You can also develop on any Posix compatible system (Linux / MacOS) in roughly t
 same way you can on a Raspberry Pi.  The emulated hardware and networking features 
 will work differently so this is mostly useful for UI/Catalog feature development.
 
+Note that you can develop on Windows by activating Windows Subsystem for Linux (WSL2) 
+and installing Ubuntu from the Microsoft Store. The window launched by PiFinder will 
+be fully integrated into your windows desktop.
```
Comment (brickbots) [inline docs/source/dev_guide.rst:99]: This is a great hint, I have little windows experience so I am very appreciative of any hints/tricks that can be offered to widen the scope of potential contributers 🙏

**Review by brickbots — CHANGES_REQUESTED** [review]

File: docs/source/dev_guide.rst (hunk 1 above)
Comment (jscheidtmann) [reply docs/source/dev_guide.rst]: Done. I rewrote the section already, see the new commit.

**Review by brickbots — APPROVED** [review]
Fantastic, great rewrite!

**Pull request merged**